gcs-action/action.yml

```diff
@@ -0,0 +1,26 @@
+name: 'GCS Action'
```

Comment:
I think it would be helpful to name this action more directly, something like 'cloud-storage-uploader`.

Reply:
Do we think we might want to cater to the download use-case in this action as well? Or else is there a compelling argument for splitting that out into a separate action?

Reply:
<opinion>
We should have separate actions for uploading and downloading
</opinion>

Reply:
Currently only targeting uploading in this action